src/main/java/frc/robot/superstructure/Superstructure.java

```diff
@@ -61,6 +69,41 @@ private boolean isNoteInElevatorRoller() {
 		return robot.getElevatorRoller().isNoteIn();
 	}
 
+	private boolean isNoteInIntake() {
+		return robot.getIntakeRoller().isNoteIn();
+	}
+
+	private boolean isReadyToShoot() {
+		boolean isFlywheelReady = robot.getFlywheel().isAtVelocity(FlywheelState.SHOOTING.getVelocity(), Tolerances.FLYWHEEL_VELOCITY_TOLERANCE);
+		// boolean isSwerveReady = swerve.isAtHeading(speaker);
```

Comment:
Why comented out?

Reply:
Because no vision so this doesn't exist😭